charts/tfy-clickhouse/values.yaml

```diff
@@ -90,7 +93,7 @@ clickhouse:
   zookeeperPort: '2181'
   labels: {}
   image:
-    repository: tfy.jfrog.io/tfy-mirror/clickhouse/clickhouse-server
+    repository: tfy-mirror/clickhouse/clickhouse-server
```

Comment:
we control this part, we use values.clickhouse.image.registery ||  .Values.Global.image.registry here now

Reply:
sure

Reply:
lets add registry field here?

Reply:
added